client/lib/analytics/ad-tracking/record-order.js

```diff
@@ -432,6 +435,66 @@ function recordOrderInGoogleAds( cart, orderId ) {
 	}
 }
 
+function recordOrderInGAEnhancedEcommerce( cart, orderId ) {
+	if ( ! isAdTrackingAllowed() ) {
+		debug( 'recordOrderInGAEnhancedEcommerce: [Skipping] ad tracking is disallowed' );
+		return;
+	}
+
+	if ( ! isGoogleAnalyticsEnabled || ! isGoogleAnalyticsEnhancedEcommerceEnabled ) {
+		debug( 'recordOrderInGAEnhancedEcommerce: [Skipping] Google Analytics is not enabled' );
+		return;
+	}
+
+	if ( 'function' !== typeof ga ) {
+		debug( 'recordOrderInGAEnhancedEcommerce: [Skipping] ga() is not defined' );
+		return;
+	}
+
+	// Create tracker
+	// eslint-disable-next-line no-undef
+	ga( 'create', TRACKING_IDS.wpcomGoogleAnalyticsGtag );
```

Comment:
This and the `require` line below could probably go [here](https://github.com/Automattic/wp-calypso/blob/a00ff0350a12bc4f4210c8ba758b63b11cbc98b2/client/lib/analytics/ad-tracking/google-analytics.js#L14) instead.